solutions/substitution_cipher.py

```diff
@@ -0,0 +1,61 @@
+"""
+Substitution Cipher
+this code provides a function to encrypt text using a caesar cipher,
+shifting lower and uppercase letter by a fixed value of 3 (as per the challenge)
+"""
+
+# Author: AYHAM
+
+import string
+
+
+def cipher(text: str) -> str:
+    """
+    Parameters:
+        text (str): The text to be encrypted.
+
+    Returns:
+        str: The encrypted text.
+
```

Comment:
you have to add the Other assumptions (eg. if it's a number, what's the expected range?)
The return value is described